elfo/benches/round_trip.rs

```diff
@@ -153,7 +153,7 @@ async fn run<const FLAGS: Flags>(
         AnyConfig::default(),
     ));
 
-    elfo::_priv::do_start(topology, |ctx, _| async move {
+    elfo::_priv::do_start(topology, false, |ctx, _| async move {
```

Comment:
Not a big fan of boolean flags in the middle of arguments, but this is a private interface anyway, so it might be okay. As an alternative, we can introduce enum for starting modes.

Reply:
flags are fine for private fns